unity/XRTProject/Assets/Scripts/Training/TrainingManager.cs

```diff
@@ -70,6 +70,7 @@ private void Awake()
         }
 
         apiService = new ApiService(token);
+        gameObject.AddComponent<Track>().Setup(apiService);
```

Comment:
@Agid-Kaharuba Couldn't figure out how to implement without Monobehaviour/using async